ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-4.8__ocp-dev-preview.yaml

```diff
@@ -0,0 +1,170 @@
+base_images:
+  ocp-4:
+    name: automation-flavors
+    namespace: stackrox
+    tag: openshift-4-stable
+  ubi-minimal:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: apollo-ci
+    namespace: stackrox
+    tag: stackrox-ui-test-0.4.9
+resources:
+  '*':
+    requests:
+      cpu: 2000m
+      memory: 4000Mi
+test_binary_build_commands: .openshift-ci/dispatch.sh test-binary-build-commands
+tests:
+- as: qa-e2e-tests
+  optional: true
+  run_if_changed: ^((generated|image|operator|pkg|make|deploy|scripts)/.*|Makefile|status\.sh|[A-Z_]+_VERSION)
+  steps:
+    env:
+      COLLECTION_METHOD: core_bpf
+      OCP_VERSION: ocp-dev-preview/candidate
```

Comment:
I keep forgetting what `ocp-dev-preview/candidate` and `ocp/candidate` means. I have to ask it every time and forget again. WDYT about adding info or links to this info to `ci-operator/config/stackrox/stackrox/README.md` or to the Confluence pages referenced from it?